docs/guidelines/content-guidelines/01-content-strategy.md

```diff
@@ -32,7 +32,7 @@ Select a title that describes the task that's accomplished, not the documented s
 With regards to structure, it’s nice to have an **introductory paragraph** ("why would I want to do this task?"), **prerequisites** if needed, then the **steps** in a numbered list, and finally the expected **result** that shows the operation was successful.
 It's good practice to have 5-9 steps; anything longer can probably be split.
 
-For all step instructions, use the [task topic template](../../../templates/resources/task.md).
+For all step instructions, use the [tutorial  template](https://github.com/kyma-project/community/blob/main/templates/resources/tutorial.md).
```

Comment:
The link doesn't work. Try: https://github.com/kyma-project/community/blob/main/templates/resources/tutorials.md
However, when I look at the template I would change the **Type** from `Tutorials` to `Tutorial` I guess 🤔